### PR TITLE
[qa] Add openwisp-utils-qa-checks

### PR DESCRIPTION
**Pull request by waleko**
Add `openwisp-utils-qa-checks` (related to openwisp/openwisp-utils#63 for this repo)

### CODE REVIEW

**Comment by waleko** [comment]
Travis is failing, as `django-models-utils` is v4.0.0, but openwisp-utils wants it to be <3.3.0 (fix it in openwisp-utils?)

File: requirements-test.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 coveralls
+openwisp-utils[qa]>=0.3.0
```
Comment (nemesifier) [inline requirements-test.txt]: use 0.3.1

**Review by nemesifier — CHANGES_REQUESTED** [review]

File: requirements-test.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 coveralls
+openwisp-utils[qa]>=0.3.1
```
Comment (atb00ker) [inline requirements-test.txt]: Lol, this might be a little frustrating but 3.2 got released, can you please update again? :joy:

**Comment by waleko** [comment]
Updated to 0.4.1, but still have the same problem:
> Travis is failing, as `django-models-utils` is v4.0.0, but openwisp-utils wants it to be <3.3.0 (fix it in openwisp-utils?)

**Pull request merged**